sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/tests/EntityBindingLiveTests.cs

```diff
@@ -0,0 +1,1120 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core.TestFramework;
+using Azure.Data.Tables;
+using Microsoft.Azure.WebJobs.Host;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Tables.Tests
+{
+    public class EntityBindingLiveTests: TablesLiveTestBase
+    {
+        private static DateTimeOffset DateTimeOffsetValue = DateTimeOffset.Parse("07-08-1997", null,  DateTimeStyles.AssumeUniversal | DateTimeStyles.AdjustToUniversal);
+        private static DateTime DateTimeValue = DateTime.Parse("07-08-1997", null,  DateTimeStyles.AssumeUniversal | DateTimeStyles.AdjustToUniversal);
+
+        public EntityBindingLiveTests(bool isAsync, bool useCosmos) : base(isAsync, useCosmos)
+        {
+        }
+
+        [RecordedTest]
+        [TestCaseSource(nameof(MethodsOf), new object[] {typeof(CanBindToNonExistingEntityProgram<>)})]
+        public async Task CanBindToNonExistingEntity(string entityType)
+        {
+            // Act & Assert
+            await CallAsync<CanBindToNonExistingEntityProgram<int>>(entityType);
+        }
+
+        private class CanBindToNonExistingEntityProgram<T>
+        {
+            public static void CustomTableEntity([Table(TableNameExpression, PartitionKey, RowKey)] CustomTableEntity<T> entity)
+            {
+                Assert.Null(entity);
+            }
+
+            public static void TableEntity([Table(TableNameExpression, PartitionKey, RowKey)] TableEntity entity)
+            {
+                Assert.Null(entity);
+            }
+
+            public static void ITableEntity([Table(TableNameExpression, PartitionKey, RowKey)] ITableEntity entity)
+            {
+                Assert.Null(entity);
+            }
+
+            public static void PocoTableEntity([Table(TableNameExpression, PartitionKey, RowKey)] PocoTableEntity<T> entity)
+            {
+                Assert.Null(entity);
+            }
+
+            public static void JObject([Table(TableNameExpression, PartitionKey, RowKey)] JObject entity)
+            {
+                Assert.Null(entity);
+            }
+        }
+
+        [RecordedTest]
+        [TestCaseSource(nameof(MethodsOf), new object[] {typeof(CanBindToExistingEntityProgram<>)})]
+        public async Task CanBindToExistingEntity(string entityType)
+        {
+            foreach (var values in AllowedTypedWithValues)
```

Comment:
This is done via the loop and not a testcase because per-test overhead becomes too great. It also avoids having 1000 recording files.